pkg/daemon/ceph/osd/volume.go

```diff
@@ -1007,6 +1010,29 @@ func GetCephVolumeRawOSDs(context *clusterd.Context, clusterInfo *client.Cluster
 			Store:         "bluestore",
 		}
 
+		if !skipDeviceClass {
+			deviceProps, err := sys.GetDeviceProperties(blockPath, context.Executor)
```

Comment:
Ideally, this could be reported by `ceph-volume`.

Reply:
@leseb yeah, that's true, but not for RAW list :( It will be something like this:

```
ceph-volume raw list
{
    "1": {
        "ceph_fsid": "bafa0f3d-6545-4bf1-9b7c-19368545f79e",
        "device": "/dev/mapper/ceph--bf0c53df--2f2a--4785--9527--82fdaac6c0be-osd--block--755d6dd5--fbaf--4ad0--863b--107a52b5cfe7",
        "osd_id": 1,
        "osd_uuid": "755d6dd5-fbaf-4ad0-863b-107a52b5cfe7",
        "type": "bluestore"
    }
}
```

So i didnt find any other way. May be you have any ideas how to do that not so tricky?

Reply:
Not really I was thinking that we could add it in ceph-volume at some point but it's fine for now.